.github/workflows/ci.yaml

```diff
@@ -68,8 +68,8 @@ jobs:
       - name: Rebuild tables
         env:
           PROFILE: cumulus-library-ci
-          WG: cumulus-deid-regression
-          DB: cumulus_deid_regression_db
+          WG: cumulus
+          DB: cumulus_regression_db
```

Comment:
Talked about in the other PR - I don't like this name. But just a reminder that if you change it there, update it here too